Cargo.toml

```diff
@@ -44,23 +44,43 @@ walkdir = "2.3.2"         # automatically test things for all files in a directo
 rand = "0.8.5"            # used for fuzz testing
 rayon = "1.5.3"           # run tests for many files in parallel
 
+[features]
+default = []
+
+# rayon is used for parallel compression
+rayon = ["dep:rayon-core"]
+
+[[example]]
+name = "7_write_raw_blocks"
+required-features = ["rayon"]
+
+[[example]]
+name = "8_read_raw_blocks"
+required-features = ["rayon"]
 
 [[bench]]
 name = "read"
 harness = false
+required-features = ["rayon"]
 
 [[bench]]
 name = "profiling"
 harness = false
+required-features = ["rayon"]
 
 [[bench]]
 name = "write"
 harness = false
+required-features = ["rayon"]
 
 [[bench]]
 name = "pixel_format_conversion"
 harness = false
+required-features = ["rayon"]
 
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
This isn't needed anymore, since rustdoc [recently added it as the default](https://github.com/rust-lang/docs.rs/pull/2390/files).